### DISCUSSION TITLE
Azure Monitor annotation/labels

### DISCUSSION THREAD

**Discussion by iculda** [grafana/grafana#67006] [Alerting]
Hello,
Can someone tell me if there is a way to get resurcename from azure metrics in alerts summary field
I tried to print all labels available by using `{{$labels}}` and i got only this
`map[__alert_rule_namespace_uid__:wf10Kv-Vk __alert_rule_uid__:lByjRbEVk alertname:EventHubNamespace Incoming messages  grafana_folder:Alerts]`
I also tried with `{{ resourcename }}` but no luck
Thank you

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
